### DISCUSSION TITLE
List or show attachments

### DISCUSSION THREAD

**Discussion by tobehn** [Budibase/budibase#2794] [Help] [answered]
I have a table with an attachments field containing images.

Can i somehow list all attachments on a page. In case its images i would like to show the attachments.
I tried URL, Image and Paragraph Element.

Thank you in advance

**Comment by mjashanks** [comment]
Hi @tobehn - I was able to list images as below - 
```
> Data Provider 
    > Repeater 
        > Image - Url binding is {{ Repeater.Table.AttachmentField }}
```

The only thing is that it only displays the first attachment for each for... not sure if showing multiple for every row is possible right now...

**Reply by tobehn** [reply]
Thank you @mjashanks this worked perfectly for the first uploaded image as you described . 

To stay in the logic it would be nice to be able to declare a specific row with its ._id as a DataProvider and get the Repeater in there to iterate over the images. I am not sure though if this makes sense.

**Comment by stephlegrand** [comment]
I am also trying to display all the attachments for one record. To me, it is important to be able to access all  of them, not just the first one.

**Reply by mjashanks** [reply]
We are currently working on this one. It should be covered by #3173

Available at the end of November.

**Reply by stephlegrand** [reply]
This is really excellent news. Congrats on this amazing work.

**Comment by mjashanks** [comment, marked as answer]
Done via https://github.com/Budibase/budibase/pull/3355
